rules/windows/process_creation/proc_creation_win_arcsoc_susp_child_process.yml

```
@@ -0,0 +1,42 @@
+title: Suspicious ArcSOC.exe Child Process
+id: 8e95e73e-ba02-4a87-b4d7-0929b8053038
+status: experimental
+description: |
+    This rule detects script interpreters, command-line tools, and similar suspicious child processes of ArcSOC.exe.
+    ArcSOC.exe is the process name which hosts ArcGIS Server REST services. If an attacker compromises an ArcGIS
+    Server system and uploads a malicious Server Object Extension (SOE), they can send crafted requests to the corresponding
+    service endpoint and remotely execute code from the ArcSOC.exe process.
+references:
+    - https://reliaquest.com/blog/threat-spotlight-inside-flax-typhoons-arcgis-compromise/
+    - https://enterprise.arcgis.com/en/server/12.0/administer/windows/inside-an-arcgis-server-site.htm
+author: Micah Babinski
+date: 2025-11-25
+tags:
+    - attack.execution
+    - attack.t1059
+    - attack.t1203
+logsource:
+    category: process_creation
+    product: windows
+detection:
+    selection:
+        ParentImage|endswith: '\ArcSOC.exe'
+        Image|endswith:
+            - '\rundll32.exe'
+            - '\regsvr32.exe'
+            - '\wscript.exe'
+            - '\cscript.exe'
+            - '\powershell.exe'
+            - '\cmd.exe'
+            - '\mshta.exe'
+    filter_cmd:
+        Image|endswith: '\cmd.exe'
+        CommandLine: 'cmd.exe /c "ver"'
```

There is a convention where if a filter is required / happens by default. It will be called a `main` filter. Hence this change.
```suggestion
    filter_main_cmd:
        Image|endswith: '\cmd.exe'
        CommandLine: 'cmd.exe /c "ver"'
```